tests/app/main/views/test_services.py

```diff
@@ -615,13 +615,23 @@ def test_service_update_with_multiquestion_validation_error(self, data_api_clien
 
         document = html.fromstring(response.get_data(as_text=True))
 
-        validation_banner = document.xpath("//div[@class='validation-masthead']//text()[normalize-space(.) != '']")
-        validation_banner_text = ' '.join([elem.strip() for elem in validation_banner])
+        validation_banner_h1 = document.xpath("//h1[@class='validation-masthead-heading']//text()")[0].strip()
+        assert validation_banner_h1 == "There was a problem with your answer to:"
+
+        validation_banner_links = [
+            (anchor.text_content(), anchor.get('href')) for anchor in
+            document.xpath("//a[@class='validation-masthead-link']")
+        ]
+        assert validation_banner_links == [
+            ("Service benefits", "#serviceBenefits"),
+            ("Service features", "#serviceFeatures")
+        ]
+
         validation_errors = [error.strip() for error in document.xpath("//span[@class='validation-message']//text()")]
-        assert response.status_code == 400
```

Comment:
Ooops - did I delete the 400 status assertion?

Reply:
All my fault I'm sure.  Will put it back.

Reply:
It's back!

Reply:
\o/